### PR TITLE
PLANNER-511 : Workbench: Solver configuration editor: replace scoreDrl field with ksessionName

### PR DESCRIPTION
**Pull request by Rikkola**
Added the ksession selector.
Split the ConfigPersistance class into two. One class makes the server side model and other class makes the model that can be used on the client side.

### CODE REVIEW

File: optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-api/src/main/java/org/optaplanner/workbench/screens/solver/model/ScoreDirectorFactoryConfigModel.java [hunk 1]
```diff
@@ -24,6 +24,8 @@
 
     protected ScoreDefinitionTypeModel scoreDefinitionType = null;
 
+    protected String ksessionName = null;
+
     protected List<String> scoreDrlList = null;
```
Comment (ge0ffrey) [inline optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-api/src/main/java/org/optaplanner/workbench/screens/solver/model/ScoreDirectorFactoryConfigModel.java:29]: Can scoreDrlList be removed entirely? We shouldn't support that at all in wb I believe.

**Comment by ge0ffrey** [comment]
+1 to merge already, we can deal with my comment later.

**Comment by csadilek** [comment]
master: 83cbcad7e3772dd21d9abad6627d62382f32b5bc

**Comment by ge0ffrey** [comment]
Any idea why JIRA says this PR is declined (which it is not)?
  https://issues.jboss.org/browse/PLANNER-511

**Comment by psiroky** [comment]
The PR was closed, so JIRA thinks it was declined. We would have to hit merge button in order to see it correctly (accepted) in JIRA. From my point of view, this is a limitation of github. We should be able to mark the PR as merged manually as well, but that is just not possible right now.

**Comment by ge0ffrey** [comment]
Ok, good to know about this limitation.

**Pull request closed without merging**